src/mongo_db_from_config/_version.py

```diff
@@ -1,2 +1,2 @@
 """This file defines the version of this module."""
-__version__ = "0.1.0"
+__version__ = "0.1.1"
```

Comment:
I consider officially adding support for new versions of Python a minor version bump (and that is what I did in the other PRs that have already been approved and merged). Thoughts?

Reply:
Please see commit 063330f.